docs/TESTS.md

```diff
@@ -1,44 +1,110 @@
-### Creating the project in Xcode
+## Xcode Tutorial
 
-Please see <a href="https://developer.apple.com/library/tvos/documentation/DeveloperTools/Conceptual/testing_with_xcode/chapters/02-quick_start.html#//apple_ref/doc/uid/TP40014132-CH2-SW1" target="_blank">Apple's guide on XCTest</a>
+This guide explains how to create and configure an Xcode project for Exercism.
 
-These steps will use the `bob` exercise as an example. If you are not working on `bob`, replace `bob` with your exercise name in the steps below.
+These instructions are written with the expectation that some readers will be very new to Xcode. Each step is described in excruciating detail. However, with a little experience performing these instructions should only take a minute.
 
```

Comment:
Your English is far from terrible. I understood what you meant. I think you did a great job of detecting weak sentences that needed to be more clear. I took your suggestions and addressed every one, although perhaps with a different emphasis. When I push the next update, I would appreciate your comments and suggestions on that draft.